### PR TITLE
Read relocation data only if the segment flags indicate they exist.

### PR DESCRIPTION
**Pull request by dmakarov**

### CODE REVIEW

File: src/ImageLoaders/MzExe/NeImageLoader.cs [hunk 1]
```diff
@@ -582,6 +582,8 @@ bool LoadSegment(NeSegment seg, MemoryArea mem, SegmentMap imageMap)
                 mem,
                 seg.Address.Selector.Value.ToString("X4"),
                 access);
+            if ((seg.Flags & 0x100) == 0)
+                return true;
             var rdr = new LeImageReader(
```
Comment (uxmal) [inline src/ImageLoaders/MzExe/NeImageLoader.cs]: Could you add a symbolic name for the "magic" 0x100 constant here?

**Comment by uxmal** [comment]
Hello Dmitri. Thanks for the PR! I added a small request for a symbolic constant -- I'm sure that will be easy for you to add.

**Comment by dmakarov** [comment]
updated.

**Comment by uxmal** [comment]
Looks good. Thanks for the contribution!

**Pull request merged**